### PR TITLE
Create PFF variable lookup tables for 2020 data and update 2010 table

### PR DESCRIPTION
**Pull request by Oysters1874**
Create new PFF variable lookup tables by copying from 2019 templates and modifying the corresponding fields based on the email from Erica. I have set the census_variable empty for all modified PFF fields, in particular, **grpi30pl** in Housing domain and **fb2010ltr** in Social domain. 
I have modified 2010 table as well to delete grpi30t34 & grpi35pl and add **grpi30pl**.

### CODE REVIEW

File: .github/workflows/acs.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ jobs:
           cat $FILE_PATH | psql $EDM_DATA -v TABLE_NAME=$TABLE_NAME -f pipelines/create_acs.sql
 
       - name: upload to s3
+        if: github.ref == 'refs/heads/dev'
```
Comment (abrieff) [inline .github/workflows/acs.yml:68]: Is this supposed to be included?

**Review by abrieff — COMMENTED** [review]
Matches changes from email, just question about the github workflow change

File: .github/workflows/acs.yml (hunk 1 above)
Comment (Oysters1874) [reply .github/workflows/acs.yml:68]: my bad, seems like I accidentally involve others' commit

Comment (Oysters1874) [reply .github/workflows/acs.yml:68]: Okay, I branched off from dev. I will modify it.

**Review by SashaWeinstein — APPROVED** [review]
These changes look good to me. We may find issues as we implement the variable mapping we can deal with those as they come up

**Review by abrieff — APPROVED** [review]

**Pull request merged**